spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/AmqpInboundPolledChannelAdapterSpec.java

```diff
@@ -37,6 +38,11 @@
 		this.target = new AmqpMessageSource(connectionFactory, queue);
 	}
 
+	public AmqpInboundPolledChannelAdapterSpec(ConnectionFactory connectionFactory,
```

Comment:
We usually don't allow to create `Spec` directly - only the factory above.
Although I wouldn't mind to start to make these ctors `protected` for the cases when we missed some property to expose but end-user still would like to use our `Spec` and extend it for omitted property.

Reply:
😄 - I let eclipse create the ctor for me

Reply:
I know. That's pain. And for me IDEA doesn't add `this.` when I generate getters 😢